platform/client-admin/.graphqlrc.yml

```diff
@@ -31,10 +31,16 @@ extensions:
           withHooks: true
           skipDocumentsValidation: true
           scalars:
+            GraphbackTime: string
+            GraphbackDate: Date
             GraphbackDateTime: Date
+            GraphbackTimestamp: number
             GraphbackObjectID: string
+            GraphbackJSON: any
+            GraphbackJSONObject: "{ [key: string]: any }"
         plugins:
-          - add: '/* tslint:disable */'
+          - add: 
+              content: '/* tslint:disable */'
```

Comment:
```suggestion
              content: '/* eslint-disable */'
```

Still using tslint?

Reply:
Yep.. Same as starter. Starter needs investment. As for OVP it will not matter much